Cargo.toml

```diff
@@ -13,6 +13,7 @@ members = [
     "shared",
     "state",
     "storage",
+    "storage-heed",
```

Comment:
I think we should keep these package inside `storage`.

Reply:
I think so too.